kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,2 @@
+# Install the NodeJS rpm (will include npm)
+rpm -i https://rpm.nodesource.com/pub_6.x/el/7/x86_64/nodejs-6.7.0-1nodesource.el7.centos.x86_64.rpm
```

Comment:
Does this also install node? Previously we were running the script that installed this rpm, but then needed to also `yum -y install nodejs`.